app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,44 @@
+class OmniauthCallbacksController < Devise::OmniauthCallbacksController
+  def facebook
+    # You need to implement the method below in your model (e.g. app/models/user.rb)
```

Comment:
When you copy code from other developers, remove their comments unless you have a really good reason to keep them.